### PR TITLE
Add missing CoAP Content-Formats

### PR DESCRIPTION
**Pull request by sophokles73**
Added the Content-Formats that have been registered with IANA but which
haven't been added to Cf yet.

### CODE REVIEW

**Comment by boaks** [comment]
2019 we had issue #1078.

So, should I ask the core-mailing list, if the content types have been consolidated before merging it?

**Comment by boaks** [comment]
[core](https://mailarchive.ietf.org/arch/msg/core/fvqzpz5iKpg4V92hzQeaWhACMxc/)

File: californium-core/src/main/java/org/eclipse/californium/core/coap/MediaTypeRegistry.java [hunk 1]
```diff
@@ -40,17 +40,58 @@
...
+	public static final int APPLICATION_COSE_KEY = 101;
+	public static final int APPLICATION_COSE_KEY_SET = 102;
 	public static final int APPLICATION_SENML_JSON = 110;
+	public static final int APPLICATION_SENSML_JSON = 111;
 	public static final int APPLICATION_SENML_CBOR = 112;
+	public static final int APPLICATION_SENSML_CBOR = 113;
+	public static final int APPLICATION_SENML_EXI = 114;
+	public static final int APPLICATION_SENSML_EXI = 115;
+	public static final int APPLICATION_COAP_GROUP = 256;
+	public static final int APPLICATION_DOTS = 271;
```
Comment (EskoDijk) [inline californium-core/src/main/java/org/eclipse/californium/core/coap/MediaTypeRegistry.java:74]: Above should be APPLICATION_DOTS_CBOR ? (See IANA registry)

File: californium-core/src/main/java/org/eclipse/californium/core/coap/MediaTypeRegistry.java [hunk 2]
```diff
@@ -40,17 +40,58 @@
...
+	public static final int APPLICATION_MISSING_BLOCKS = 272;
```
Comment (EskoDijk) [inline californium-core/src/main/java/org/eclipse/californium/core/coap/MediaTypeRegistry.java:75]: Also here add '_CBOR_SEQ' to name?

**Pull request merged**